### PR TITLE
R outsidefiles 1

### PR DESCRIPTION
**Pull request by jerrymae1999**

### CODE REVIEW

File: contrib/devtools/copyright_header.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def compile_copyright_regex(copyright_style, year_style, name):
 
 EXPECTED_HOLDER_NAMES = [
     r"Satoshi Nakamoto",
-    r"The Bitcoin Core developers",
+    r"The Revolt core developers",
```
Comment (kathy129) [inline contrib/devtools/copyright_header.py:96]: will you please change "core" to "Core" for uniformity?

File: contrib/devtools/copyright_header.py [hunk 2]
```diff
@@ -334,7 +334,7 @@ def write_file_lines(filename, file_lines):
 COPYRIGHT = r'Copyright \(c\)'
 YEAR = "20[0-9][0-9]"
 YEAR_RANGE = '(%s)(-%s)?' % (YEAR, YEAR)
-HOLDER = 'The Bitcoin Core developers'
+HOLDER = 'The Revolt core developers'
```
Comment (kathy129) [inline contrib/devtools/copyright_header.py:337]: same comment from above

File: contrib/devtools/gen-manpages.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-# Copyright (c) 2022 The Bitcoin Core developers
+# Copyright (c) 2022 The Revolt core developers
```
Comment (kathy129) [inline contrib/devtools/gen-manpages.py:2]: same here

File: contrib/devtools/security-check.py [hunk 4]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-# Copyright (c) 2015-2021 The Bitcoin Core developers
+# Copyright (c) 2015-2021 The Revolt core developers
```
Comment (kathy129) [inline contrib/devtools/security-check.py:2]: same here

File: contrib/devtools/test-security-check.py [hunk 5]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-# Copyright (c) 2015-2021 The Bitcoin Core developers
+# Copyright (c) 2015-2021 The Revolt core developers
```
Comment (kathy129) [inline contrib/devtools/test-security-check.py:2]: here

File: contrib/devtools/test-symbol-check.py [hunk 6]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-# Copyright (c) 2020-2021 The Bitcoin Core developers
+# Copyright (c) 2020-2021 The Revolt core developers
```
Comment (kathy129) [inline contrib/devtools/test-symbol-check.py:2]: also here

File: contrib/devtools/test_deterministic_coverage.sh [hunk 7]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 #
-# Copyright (c) 2019-2020 The Bitcoin Core developers
+# Copyright (c) 2019-2020 The Revolt core developers
```
Comment (kathy129) [inline contrib/devtools/test_deterministic_coverage.sh:3]: here.

File: contrib/devtools/utils.py [hunk 8]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-# Copyright (c) 2021 The Bitcoin Core developers
+# Copyright (c) 2021 The Revolt core developers
```
Comment (kathy129) [inline contrib/devtools/utils.py:2]: here

File: contrib/devtools/utxo_snapshot.sh [hunk 9]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 #
-# Copyright (c) 2019 The Bitcoin Core developers
+# Copyright (c) 2019 The Revolt core developers
```
Comment (kathy129) [inline contrib/devtools/utxo_snapshot.sh:3]: here

File: contrib/filter-lcov.py [hunk 10]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-# Copyright (c) 2017-2020 The Bitcoin Core developers
+# Copyright (c) 2017-2020 The Revolt core developers
```
Comment (kathy129) [inline contrib/filter-lcov.py:2]: here.

File: contrib/guix/README.md [hunk 11]
```diff
@@ -1,4 +1,4 @@
-# Bootstrappable Bitcoin Core Builds
+# Bootstrappable Revolt core Builds
```
Comment (kathy129) [inline contrib/guix/README.md:1]: here.

File: contrib/guix/README.md [hunk 12]
```diff
@@ -57,7 +57,7 @@ and examples](#common-guix-build-invocation-patterns-and-examples) section below
 before starting a build. For a full list of customization options, see the
 [recognized environment variables][env-vars-list] section.*
 
-To build Bitcoin Core reproducibly with all default options, invoke the
+To build Revolt core reproducibly with all default options, invoke the
```
Comment (kathy129) [inline contrib/guix/README.md:60]: here

File: contrib/guix/README.md [hunk 13]
```diff
@@ -80,7 +80,7 @@ crucial differences:
     * _**DETACHED_SIGS_REPO**_
 
       Set the directory where detached codesignatures can be found for the current
-      Bitcoin Core version being built.
+      Revolt core version being built.
```
Comment (kathy129) [inline contrib/guix/README.md:83]: here

File: contrib/guix/guix-attest [hunk 14]
```diff
@@ -3,7 +3,7 @@ export LC_ALL=C
 set -e -o pipefail
 
 # Source the common prelude, which:
-#   1. Checks if we're at the top directory of the Bitcoin Core repository
+#   1. Checks if we're at the top directory of the Revolt core repository
```
Comment (kathy129) [inline contrib/guix/guix-attest:6]: here.

File: contrib/guix/guix-build [hunk 15]
```diff
@@ -3,7 +3,7 @@ export LC_ALL=C
 set -e -o pipefail
 
 # Source the common prelude, which:
-#   1. Checks if we're at the top directory of the Bitcoin Core repository
+#   1. Checks if we're at the top directory of the Revolt core repository
```
Comment (kathy129) [inline contrib/guix/guix-build:6]: here

File: contrib/guix/guix-build [hunk 16]
```diff
@@ -157,7 +157,7 @@ done
 if (( total_required_KiB > avail_KiB )); then
     total_required_GiB=$((total_required_KiB / 1048576))
     avail_GiB=$((avail_KiB / 1048576))
-    echo "Performing a Bitcoin Core Guix build for the selected HOSTS requires ${total_required_GiB} GiB, however, only ${avail_GiB} GiB is available. Please free up some disk space before performing the build."
+    echo "Performing a Revolt core Guix build for the selected HOSTS requires ${total_required_GiB} GiB, however, only ${avail_GiB} GiB is available. Please free up some disk space before performing the build."
```
Comment (kathy129) [inline contrib/guix/guix-build:160]: here

File: contrib/guix/guix-clean [hunk 17]
```diff
@@ -3,7 +3,7 @@ export LC_ALL=C
 set -e -o pipefail
 
 # Source the common prelude, which:
-#   1. Checks if we're at the top directory of the Bitcoin Core repository
+#   1. Checks if we're at the top directory of the Revolt core repository
```
Comment (kathy129) [inline contrib/guix/guix-clean:6]: please continue to recheck each of the files under this folder. Thank you.

**Review by kathy129 — CHANGES_REQUESTED** [review]
please continue to recheck each of the files under this folder. I have made some changes and left some comments as well for you to work on. Thank you.

File: contrib/guix/guix-clean (hunk 17 above)
Comment (jerrymae1999) [reply contrib/guix/guix-clean]: Done changing core to Core

File: contrib/guix/guix-build (hunk 16 above)
Comment (jerrymae1999) [reply contrib/guix/guix-build]: Done changing core to Core

File: contrib/guix/guix-build (hunk 15 above)
Comment (jerrymae1999) [reply contrib/guix/guix-build]: Done changing core to Core

File: contrib/guix/guix-attest (hunk 14 above)
Comment (jerrymae1999) [reply contrib/guix/guix-attest]: Done changing core to Core

File: contrib/guix/README.md (hunk 13 above)
Comment (jerrymae1999) [reply contrib/guix/README.md]: Done changing core to Core

File: contrib/guix/README.md (hunk 12 above)
Comment (jerrymae1999) [reply contrib/guix/README.md]: Done changing core to Core

File: contrib/guix/README.md (hunk 11 above)
Comment (jerrymae1999) [reply contrib/guix/README.md]: Done changing core to Core

File: contrib/filter-lcov.py (hunk 10 above)
Comment (jerrymae1999) [reply contrib/filter-lcov.py]: Done changing core to Core

File: contrib/devtools/utxo_snapshot.sh (hunk 9 above)
Comment (jerrymae1999) [reply contrib/devtools/utxo_snapshot.sh]: Done changing core to Core

File: contrib/devtools/utils.py (hunk 8 above)
Comment (jerrymae1999) [reply contrib/devtools/utils.py]: Done changing core to Core

File: contrib/devtools/test_deterministic_coverage.sh (hunk 7 above)
Comment (jerrymae1999) [reply contrib/devtools/test_deterministic_coverage.sh]: Done changing core to Core

File: contrib/devtools/test-symbol-check.py (hunk 6 above)
Comment (jerrymae1999) [reply contrib/devtools/test-symbol-check.py]: Done changing core to Core

File: contrib/devtools/test-security-check.py (hunk 5 above)
Comment (jerrymae1999) [reply contrib/devtools/test-security-check.py]: Done changing core to Core

File: contrib/devtools/security-check.py (hunk 4 above)
Comment (jerrymae1999) [reply contrib/devtools/security-check.py]: Done changing core to Core

**Review by kathy129 — APPROVED** [review]
Reviewed.

**Pull request merged**